entrypoint.sh

```diff
@@ -137,6 +144,11 @@ echo "[+] Set directory is safe ($CLONE_DIR)"
 # TODO: review before releasing it as a version
 git config --global --add safe.directory "$CLONE_DIR"
 
+
+if [ "$CREATE_TARGET_BRANCH_IF_NEEDED" = 'true' ] ; then
+  git checkout -b "$TARGET_BRANCH"
+fi
```

Comment:
Create our branch if needed before adding and committing files.

We do not set the upstream as this part was already covered by the `push` command:
```sh
echo "[+] Pushing git commit"
# --set-upstream: sets de branch when pushing to a branch that does not exist
git push "$GIT_CMD_REPOSITORY" --set-upstream "$TARGET_BRANCH"
```